### PR TITLE
Allow addIceCandidate(null) again for backwards comp and symmetry.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fix for consideration for https://github.com/w3c/webrtc-pc/issues/2039.

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webrtc-pc/pull/2040.html" title="Last updated on Dec 11, 2018, 3:32 PM GMT (1f50e71)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2040/66678f6...jan-ivar:1f50e71.html" title="Last updated on Dec 11, 2018, 3:32 PM GMT (1f50e71)">Diff</a>

### CODE REVIEW

**Comment by lgrahl** [comment]
@jan-ivar If there isn't a test for this in WPT, we need one. :slightly_smiling_face:

File: webrtc.html [hunk 1]
```diff
@@ -2882,7 +2882,8 @@ <h2>Methods</h2>
                     method was invoked.</p>
                   </li>
                   <li>
-                    <p>If both <var>sdpMid</var> and <var>sdpMLineIndex</var> are
+                    <p>If <var>candidate</var> is not an empty string and both
+                    <var>sdpMid</var> and <var>sdpMLineIndex</var> are
```
Comment (henbos) [inline webrtc.html]: All of these should probably say candidate.foo. The line above is "let candidate be the method's first argument"

File: webrtc.html [hunk 2]
```diff
@@ -3012,6 +3013,10 @@ <h2>Methods</h2>
                     </ol>
                   </li>
                 </ol>
+                <p class="note">In the above algorithm, addIceCandidate(null)
+                succeeds, indicating end-of-candidates for all media
```
Comment (henbos) [inline webrtc.html]: Weird sentence, should it say "use addIceCandidate(null) to indicate end-of-candidates" or "if addIceCandidate(null) succeeds..."? What does it mean to indicate end of ICE candidate generation in the context of adding?

**Comment by aboba** [comment]
@jan-ivar The PR doesn't cite JSEP.

**Pull request merged**